templates/style_guides/javascript.json

```diff
@@ -0,0 +1,3 @@
+{
+  "camelcase": true
```

Comment:
Sorry for my ignorance, but isn't that activating camelcase instead of snake case for JS ?